{{cookiecutter.project_slug}}/.github/workflows/memote.yml

```diff
@@ -0,0 +1,30 @@
+name: memote
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+          fetch-depth: 0
+    # - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install memote==0.13.0
+    - name: Run memote on deployment branch
+      env:
+        GITHUB_TOKEN: {{ '${{ secrets.GITHUB_TOKEN }}' }}
```

Comment:
Perhaps, instead of requiring a token, we can get away with [setting write permissions for this job](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)?

Reply:
I think I got it right (tested at https://github.com/carrascomj/ecoli_test_permissions/actions/runs/6202554148/job/16841522271), but I had not work with permissions this way before.